src/api/public/apidocs-new/paths/source_project_name_package_name_binary_filename_attribute_name.yaml

```diff
@@ -0,0 +1,53 @@
+get:
+  summary: Get list of attributes and attribute values
+  description: |
+    Get a list of all attributes and their values or a list of attribute values when attribute is provided in parameter
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: path
+      name: package_name
+      schema:
+        type: string
+      description: Package name
+      example: "ctris"
+    - in: path
+      name: binary_filename
+      schema:
+        type: string
+      description: Binary filename
+      example: ctris-0.42.1-8.1.x86_64.rpm
+    - in: path
+      name: attribute_name
+      schema:
+        type: string
+      description: The name of the attribute with namespace
+      example: OBS:OwnerRootProjectTest
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/attributes.yaml'
+          example:
+            name: AllowSubmitToMaintenanceRelease
+            namespace: OBS
+    '400':
+      description: Bad request
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            invalid_attribute:
+              value:
+                code: 400
```

Comment:
I'd use `invalid_attribute` instead of `400`, as the response is:

```
<status code="invalid_attribute">
  <summary>Attribute 'ScreenShots' must be in the $NAMESPACE:$NAME style</summary>
</status>
```

Reply:
Done